poc/vdaf_prio3.py

```diff
@@ -323,14 +348,15 @@ def joint_rand_seed(Prio3, k_joint_rand_parts):
         )
 
     @classmethod
-    def joint_rand(Prio3, k_joint_rand_seed):
+    def joint_rands(Prio3, k_joint_rand_seed):
         """Derive the joint randomness from its seed."""
+        binder = b'' if Prio3.PROOFS == 1 else byte(Prio3.PROOFS)
```

Comment:
Choose to be backward compatible, i.e. only embedding `Prio3.PROOFS` when it is > 1.

Reply:
Yup, good idea.

Reply:
Reviewer note: I've confirmed the test vectors haven't changed.